pkg/reconciler/topology/partitionset/resources.go

```diff
@@ -51,11 +51,11 @@ func generatePartitionName(name string, matchLabels map[string]string, dimension
 	for _, label := range labels {
 		name = name + "-" + strings.ToLower(matchLabels[label])
 	}
-	name = name[:min(validation.DNS1123SubdomainMaxLength-1, len(name))]
+	name = name[:minimum(validation.DNS1123SubdomainMaxLength-1, len(name))]
 	return name
 }
 
-func min(a, b int) int {
+func minimum(a, b int) int {
```

Comment:
can we just remove our `min`? It's not in the language.

Reply:
Good point. I forgot that there is the builtin `min` now ...